src/portfolio.js

```diff
@@ -238,6 +238,11 @@ const bigProjects = {
   display: true, // Set false to hide this section, defaults to true
 };
 
+const githubActivity = {
+  userName: 'YOUR_USERNAME', // Change to your codersRank username to display widgets from codersRank
```

Comment:
@akashp1712 change it saadpasta so we can view the activity. 
Q: Do I need to have codersRank profile to view the Github Activity

Reply:
Yes @saadpasta, you will require to have the **codersrank** account. As they are rendering the GitubActivity component.

Reply:
If it won't work without a profile on that application then it isn't really user friendly imo. Since we're already asking for Github token from users, we can use an open source GitHub graph component (if there is any) to save the trouble of making an account for this feature. Let me know what you think.

Reply:
@akashp1712 would agree with @kartikcho could you look up some other library for this.

Reply:
Yeah, agree with this as not all user would go and make an account in the third-party apps.
This was the only available solution with interactive github activity widget (afaik). We can close this pull until we find something similar.

Reply:
I'd really hate for your work to go to waste but this could really be a great addition next to our GitHub projects section. If you could find any alternatives that work using gh access tokens, definitely open a PR for adding it (or an issue first to make sure it doesn't go to waste).

I'll close it for now but feel free to look into other issues if you want to dabble more with React!